src/battle_message.c

```diff
@@ -363,8 +363,11 @@ static const u8 sText_SunlightGotBright[] = _("The sunlight got bright!");
 static const u8 sText_SunlightStrong[] = _("The sunlight is strong.");
 static const u8 sText_SunlightFaded[] = _("The sunlight faded.");
 static const u8 sText_StartedHail[] = _("It started to hail!");
+static const u8 sText_StartedSnow[] = _("It started to snow!");
 static const u8 sText_HailContinues[] = _("Hail continues to fall.");
+static const u8 sText_SnowContinues[] = _("Snow continues to fall.");
 static const u8 sText_HailStopped[] = _("The hail stopped.");
+static const u8 sText_SnowStopped[] = _("The snow stopped.");
```

Comment:
I would put the Snow messages together instead of intercalating them with the hail ones